src/explicitode.jl

```diff
@@ -84,6 +82,21 @@ function jetcoeffs!(eqsdiff!, t::Taylor1{T}, x::AbstractVector{Taylor1{U}},
     nothing
 end
 
+
+# __jetcoeffs
+"""
+    __jetcoeffs!(::Val{bool}, f, t, x)
+    __jetcoeffs!(::Val{bool}, f, t, x, dx, xaux)
+
+Chooses a method of [`jetcoeffs!`](@ref) (hard-coded or generated by
+[`@taylorize`](@ref)) depending on `bool::Bool`.
+"""
+@inline __jetcoeffs!(::Val{false}, f, t, x) = jetcoeffs!(f, t, x)
```

Comment:
@lbenet this is really neat! Just great that you figured out how to select the different cases actually in a Julian way of doing things!